source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -192,6 +192,116 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+Conditional Expression Operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can perform conditional operations using the methods described in
+this section.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
+   * - `cond() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlBoolean.html#cond(T,T)>`__
+     - :manual:`$cond </reference/operator/aggregation/cond/>`
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+
+Suppose you want to find all documents where a customer purchased
+enough to qualify for a discount. The amount they need to spend for a discount
+varies if they are a team member.
+
+To decide whether the customer is a team member, check
+``isTeamMember`` with the ``cond()`` method. To determine whether the sale
+was large enough, check the ``totalSale`` with the ``gte()`` method.
+
+The following code shows the pipeline stage for this aggregation:
+
+.. code-block:: java
+   :copyable: true
+
+   Arrays.asList(match(
+            expr(current().getBoolean("isTeamMember").cond(
+                     current().getNumber("totalSale")
+                        .gte(of(100)),
+                     current().getNumber("totalSale")
+                        .gte(of(200))
+            ))
+   ));
+
+The following code provides an equivalent aggregation pipeline in
+the Query API:
+
+.. code-block:: javascript
+   :copyable: true
+
+   [
+     {
+       $match: {
+         $cond: {
+            "$isTeamMember",
+            { $gte: [ "$totalSale", 100 ] },
+            { $gte: [ "$totalSale", 200 ] }
+         }
+       }
+     }
+   ]
+
+Set Expression Operations
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set expressions perform set operations on values of type ``MqlArray``, treating
+the arrays as sets.
```

Comment:
Does this sound ok? Reworded to use "You can..." for consistency

```suggestion
You can perform set operations on values of type ``MqlArray``, as these operations treat
the arrays as sets.
```

Reply:
I broke it up into two sentences because it read better to me that way.